### PR TITLE
Bugfix/dropdown and pagination

### PR DESCRIPTION
**Pull request by d-suman1**
Issues resolved:

1. search input was getting cleared if value was falsy and items is  updated
2. fixed dropdown appearance issue when there is not enough bottom space
3. added new props for tooltip position and icon in dropdown

### CODE REVIEW

File: src/components/Dropdown/Dropdown.scss [hunk 1]
```diff
@@ -88,8 +88,9 @@
   }
 
   .n-dropup {
-    transform: translate3d(0rem, 0rem, 0rem);
-    bottom: 4.2rem;
+    transform: translate3d(0rem, -4.2rem, 0rem);
+    bottom: 18px;
```
Comment (ShahRohan27) [inline src/components/Dropdown/Dropdown.scss:92]: use rem

**Review by ShahRohan27 — COMMENTED** [review]
Please add test cases

**Comment by d-suman1** [comment]
@ShahRohan27 I believe I have already added a test case for the Dropdown issue resolved here. Could you clarify on which part testcases are required?

File: src/components/Dropdown/Dropdown.scss (hunk 1 above)
Comment (d-suman1) [reply src/components/Dropdown/Dropdown.scss]: Done

**Pull request merged**